### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by pavel-gutsal**
https://pavel-gutsal.github.io/layout_catalog_hovers/

### CODE REVIEW

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 0.3s ease-in-out;
```
Comment (ArturErmolenko) [inline src/styles/_catalog.scss:17]: Лучше явно указывать свойство которое анимируеш
```suggestion
    transition: transform 0.3s ease-in-out;
```

File: src/styles/_header.scss [hunk 2]
```diff
@@ -25,17 +25,29 @@
...
 
   &__link {
+    display: flex;
+    height: 100%;
     color: inherit;
     text-decoration: none;
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: all 0.3s ease-in-out;
```
Comment (ArturErmolenko) [inline src/styles/_header.scss:43]: same

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]

**Comment by pavel-gutsal** [comment]
transition values were changed to specific changes.

**Review by ArtemTopolskyi — CHANGES_REQUESTED** [review]

**Comment by pavel-gutsal** [comment]
transition-property: transform , color;             was added to class

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -14,6 +13,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 0.3s ease-in-out;
+    transition-property: transform , color ;
```
Comment (Bohdan-Maksymiuk) [inline src/styles/_catalog.scss:17]:
```suggestion
    transition-property: transform, color;
```

**Review by Bohdan-Maksymiuk — APPROVED** [review]

**Pull request closed without merging**